src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -201,6 +202,7 @@ def _get_subprocess_kwargs_for_fluent(env: Dict[str, Any]) -> Dict[str, Any]:
     fluent_env = os.environ.copy()
     fluent_env.update({k: str(v) for k, v in env.items()})
     fluent_env["REMOTING_THROW_LAST_TUI_ERROR"] = "1"
+    fluent_env["FLUENT_LAUNCHED_FROM_PYFLUENT"] = "1"
```

Comment:
Need to skip the password as currently there is no way to print the password to stdout